src/cuga/backend/server/main.py

```
@@ -1005,6 +1021,57 @@ async def _save_conversation_and_events_async(
+            app_state.stop_events[tid].clear()
+
+    try:
+        return await parse_and_dispatch(
+            query,
+            slash_registry=slash_registry,
+            skill_registry=skill_registry,
+            thread_id=thread_id,
+            clear_stop_event=_clear_stop_event,
+            command_resolver_factory=build_command_resolver,
+        )
+    except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)